libs/autofile/autofile.go

```diff
@@ -30,7 +31,17 @@ if err != nil {
 }
 */
 
-const autoFileOpenDuration = 1000 * time.Millisecond
+const (
+	autoFileOpenDuration = 1000 * time.Millisecond
+	autoFilePerms        = os.FileMode(0600)
+)
+
+// PermissionsChangedErr occurs if the file permission have changed since the file was created.
+type PermissionsChangedErr string
```

Comment:
Prefixing of `Err` is the de facto standard, compared to suffxing: `PermissionsChangedErr` -> `ErrPermissionsChanged`.